std/algorithm/sorting.d

```diff
@@ -774,8 +774,54 @@ private template validPredicates(E, less...)
             && validPredicates!(E, less[1 .. $]);
 }
 
+struct MultiPred( preds... )
+if( preds.length >= 1 )
+{
+    private import std.functional : binaryFun;
+
+    static bool opCall( A, B )( A a, B b ) {
+        foreach( pred; preds[ 0 .. ( $ - 1 ) ] ) {
```

Comment:
No need for parens around `$ - 1`, spacing is a bit "weird" as well.
I'd highly recommend [dfmt](https://github.com/Hackerpilot/dfmt) and stop caring about syntax.